pkgs/development/python-modules/imagededup/default.nix

```diff
@@ -0,0 +1,84 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchurl
+, cython
+, torch
+, torchvision
+, pillow
+, tqdm
+, scikit-learn
+, pywavelets
+, matplotlib
+, pytest
+, pytest-mock
+}:
+let
+  MobileNetV3 = fetchurl {
+    url = "https://download.pytorch.org/models/mobilenet_v3_small-047dcff4.pth";
+    hash = "sha256-BH3P9K3e+G6lvC7/E8lhTcEfR6sRYNCnGiXn25lPTh8=";
+  };
+  ViT = fetchurl {
+    url = "https://download.pytorch.org/models/vit_b_16_swag-9ac1b537.pth";
+    hash = "sha256-msG1N42ZJ71sg3TODNVX74Dhs/j7wYWd8zLE3J0P2CU=";
+  };
+  EfficientNet = fetchurl {
+    url = "https://download.pytorch.org/models/efficientnet_b4_rwightman-7eb33cd5.pth";
+    hash = "sha256-I6uLzVvb72GnpDuRrcrYH2Iv1/NvtJNaVpgo13iIxE4=";
+  };
```

Comment:
Can we get the paths for the symlinks below from here somehow?

Reply:
We can do the other way around and reuse the filenames from here in the linking below. Is this fine for you?

Reply:
I think I mean to write that